commit ed80e93b7277df558c87e996e723a3a6130a5c09 (refs/changes/18/3118/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-12-17T16:38:41-08:00 (1 year, 10 months ago)
    
    HEAD-2383 'sdc-usbkey update' was broken by HEAD-2380, which breaks CN setup

diff --git a/tools/lib/usbkey.js b/tools/lib/usbkey.js
index f684a46a..fd358b54 100644
--- a/tools/lib/usbkey.js
+++ b/tools/lib/usbkey.js
@@ -656,8 +656,8 @@ ensure_usbkey_mounted(options, callback)
         'options.alt_mount_options');
     mod_assert.func(callback, 'callback');
 
-    dprintf('ensuring usb key is mounted (altmountopts: %s)...\n',
-        JSON.stringify(options.alt_mount_options));
+    dprintf('ensuring usb key is mounted (altmountopts: %j)...\n',
+        options.alt_mount_options);
 
     var epoch = process.hrtime();
     var mtpt;
